client/my-sites/checkout/checkout-thank-you/atomic-store-thank-you-card.jsx

```diff
@@ -32,6 +33,10 @@ const RESEND_SUCCESS = 'RESEND_SUCCESS';
 class AtomicStoreThankYouCard extends Component {
 	state = { resendStatus: RESEND_NOT_SENT };
 
+	componentDidMount() {
+		this.props.showMasterbar();
+	}
+
```

Comment:
When the AtomicStoreThankYouCard component (modal) is mounted, show the masterbar. Or in our case, reappear the masterbar.